packages/test/test-end-to-end-tests/src/test/attachRegisterLocalApiTests.spec.ts

```diff
@@ -71,7 +71,7 @@ describe(`Attach/Bind Api Tests For Attached Container`, () => {
             urlResolver,
             documentServiceFactory,
             codeLoader,
-            logger: ChildLogger.create(getTestLogger(), undefined, {all: {testDriverType: driver.type}}),
+            logger: ChildLogger.create(getTestLogger(), undefined, {all: {hostName: `end-to-end tests (${driver.type})`}}),
```

Comment:
This seems like it could go in a helper in test-utils, but I would want to get the missing hooks sorted out (on the table tests that use local loader) so that helper doesn't need to do the `?.`.

Reply:
Or swap in the child logger in a new mocha hook for e2e tests.  The thing is, I also want to put the test case name on the child logger, and I can't figure out how to do that in a hook, so if that's not possible then we'll be adding another ChildLogger in the test code anyway.

Reply:
Will follow up on these ideas elsewhere.